sdk/webpubsub/azure-messaging-webpubsub/src/main/java/com/azure/messaging/webpubsub/models/WebPubSubContentType.java

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.messaging.webpubsub.models;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+/** Defines values for ContentType. */
+public enum WebPubSubContentType {
```

Comment:
Would this be something we want to support in azure-core?

Reply:
This was discussed during arch board and it was decided to keep it here as this enum would only have the list of content-types supported by this service.